manim/mobject/types/vectorized_mobject.py

```diff
@@ -1213,6 +1213,57 @@ def __setitem__(self, key, value):
             self.remove(key)
         self.add([(key, value)])
 
+    def __delitem__(self, key):
+        """Override the del operator for deleting an item.
```

Comment:
Question: Would it be a good idea to mention that it does not remove the mobject associated with the key from the scene?

Reply:
Good point. Done.